docs/source/guides/user/chapters/concepts.rst

```diff
@@ -24,12 +24,20 @@ location::
   JOB LOG    : $HOME/avocado/job-results/job-2015-06-10T10.44-49ec339/job.log
 
 
+Test Resolver
+~~~~~~~~~~~~~
+
+A test resolver is Avocado's component that will take a reference you
+know about and turning into an actual test that can be run.  This
```

Comment:
IMO, there is a typo. `know about and turning into`/` know about and turns it into`. But my English knowledge is not perfect, therefore I might be wrong.

Reply:
You're right! I'm sending an updated version.